integration/raft/channel_participation_test.go

```diff
@@ -731,6 +731,161 @@ var _ = Describe("ChannelParticipation", func() {
+				Status:            "active",
+				ConsensusRelation: "follower",
+				Height:            6,
+			}))
+
+			By("removing orderer1 from a channel")
+			channelparticipation.Remove(network, orderer1, "participation-trophy")
+			Eventually(func() channelparticipation.ChannelList {
+				return channelparticipation.List(network, orderer1)
+			}, network.EventuallyTimeout).Should(Equal(channelparticipation.ChannelList{
+				SystemChannel: nil,
+				Channels:      nil,
+			}))
+
+			By("submitting transaction to orderer1")
+			env := CreateBroadcastEnvelope(network, peer, "participation-trophy", []byte("hello"))
+			resp, err := ordererclient.Broadcast(network, orderer1, env)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.Status).To(Equal(common.Status_BAD_REQUEST))
+
+			By("ensuring the channel is still usable by submitting a transaction to each remaining consenter for the channel")
+			submitPeerTxn(orderer2, peer, network, channelparticipation.ChannelInfo{
+				Name:              "participation-trophy",
+				URL:               "/participation/v1/channels/participation-trophy",
+				Status:            "active",
+				ConsensusRelation: "consenter",
+				Height:            7,
+			})
+
+			submitPeerTxn(orderer3, peer, network, channelparticipation.ChannelInfo{
+				Name:              "participation-trophy",
+				URL:               "/participation/v1/channels/participation-trophy",
+				Status:            "active",
+				ConsensusRelation: "consenter",
+				Height:            8,
+			})
+
+			By("joining orderer1 to channel it was previously removed from as consenter")
+			configBlockPT := nwo.GetConfigBlock(network, peer, orderer2, "participation-trophy")
+			expectedChannelInfo := channelparticipation.ChannelInfo{
+				Name:              "participation-trophy",
+				URL:               "/participation/v1/channels/participation-trophy",
+				Status:            "onboarding",
+				ConsensusRelation: "follower",
+				Height:            0,
+			}
+			channelparticipation.Join(network, orderer1, "participation-trophy", configBlockPT, expectedChannelInfo)
+
+			By("ensuring orderer3 completes onboarding successfully")
+			expectedChannelInfo = channelparticipation.ChannelInfo{
+				Name:              "participation-trophy",
+				URL:               "/participation/v1/channels/participation-trophy",
+				Status:            "active",
+				ConsensusRelation: "consenter",
+				Height:            8,
+			}
+			Eventually(func() channelparticipation.ChannelInfo {
+				return channelparticipation.ListOne(network, orderer3, "participation-trophy")
+			}, network.EventuallyTimeout).Should(Equal(expectedChannelInfo))
+
+			submitPeerTxn(orderer3, peer, network, channelparticipation.ChannelInfo{
+				Name:              "participation-trophy",
+				URL:               "/participation/v1/channels/participation-trophy",
+				Status:            "active",
+				ConsensusRelation: "consenter",
+				Height:            9,
+			})
+		})
```

Comment:
Testing that onboarding completes should be performed against orderer1, which just joined, right?
Then test orderer1 is a follower, picking up the latest block, turn orderer1 into a consenter (member), and submit a tx to it as well?

Reply:
BTW, is there a test case where we delete a channel from an orderer while it is a consenter, not a follower? i.e first delete the channel from ordererX, and only then remove ordererX from the consenters set of that channel?

Reply:
As @wlahti noted, this test is based off another and I forget to switch to orderer1 instead as he suggested I'm going to add to the existing test. Also no, we don't have a test where we delete while the orderer is a consenter.